packages/strapi-plugin-users-permissions/config/schema.graphql.js

```diff
@@ -53,18 +53,18 @@ module.exports = {
       user: UsersPermissionsMe!
     }
 
-    type ForgotPassword {
-      ok: Boolean
+    type UserPersmissionsPasswordPayload {
+      ok: Boolean!
     }
   `,
   query: `
     me: UsersPermissionsMe
   `,
   mutation: `
     login(input: UsersPermissionsLoginInput!): UsersPermissionsLoginPayload!
-    register(input: UsersPermissionsRegisterInput!): UsersPermissionsLoginPayload!
-    forgotPassword(email: String!): ForgotPassword
-    changePassword(password: String!, passwordConfirmation: String!, code: String!): UsersPermissionsLoginPayload
+    register(input: UserInput!): UsersPermissionsRegisterInput!
```

Comment:
You forgot the input too :D `UsersPermissionsRegisterInput `

Reply:
ahh lol ebed726d7